lib/ansible/modules/user.py

```diff
@@ -86,12 +86,13 @@
         version_added: "2.0"
     password:
         description:
-            - Optionally set the user's password to this encrypted value.
-            - On macOS systems, this value has to be cleartext. Beware of security issues.
+            - Optionally set the user password.
```

Comment:
- If provided, set the user's password to the provided encrypted hash (linux/unix/POSIX) or plain text password (OS X/Mac OS)

Reply:
Isn't that redundant with the text below?

Also, I don't think that Linux is "more Unix" than macOS :)

Reply:
why its in list of linux/unix/posix otherwise it woudl be implied under unix

Reply:
and yes, it makes other lines redundant, but i think its simple, concise and clear in one sentence and will avoid people 'trailing off' and not reading a wall of text.